lib/runner/concurrency/child-process.js

```diff
@@ -171,7 +170,7 @@ class ChildProcess extends EventEmitter {
 
           let status = code > 0 ? symbols.fail : symbols.ok;
           // eslint-disable-next-line no-console
-          console.log(boxen(this.env_output.join('\n'), {title: `────────────────── ${status} ${this.env_label}`, padding: 1, borderColor: 'cyan'}));
+          console.log(this.env_output.join('\n'), `\n────────── ${status} ${this.env_label}`);
```

Comment:
maybe we should add an extra `\n` at the end as well

Reply:
also, can we move the `status` and  `env_label` to the front and remove -----? It looks much cleaner this way: 
<img width="817" alt="Screenshot 2023-02-20 at 9 09 29 PM" src="https://user-images.githubusercontent.com/28780767/220149145-9980aea5-ba37-4d05-87bf-e36c27d6d2a6.png">

Reply:
```suggestion
          console.log(`\n${status} ${this.env_label}`, this.env_output.join('\n'), '\n');
```